payments/models.py

```diff
@@ -295,6 +298,51 @@ class Meta:
     def __str__(self):
         return f"{self.product} [{self.status}]"
 
+    @property
+    def order_type(self) -> OrderType:
```

Comment:
Should this be a DB enum column instead? The db column would be more future proof, performant and would enable searching by order_type in case we want it in upcoming "invoices list" view. We can of course refactor this to DB column later.

Reply:
I think the ideal approach would be to add it as an annotation (like we have done with available berths or active leases), but I think it will be a bit complex, so it will take some time to implement it.